pkgs/development/tools/analysis/retdec/default.nix

```diff
@@ -92,8 +105,7 @@ let
     SUPPORT_PKG = retdec-support;
   } // lib.optionalAttrs enableTests {
     KEYSTONE = keystone;
-    # nixpkgs googletest is used
-    # GOOGLETEST = googletest;
+    GOOGLETEST = gtest.src // { rev = ""; };  # empty rev bypasses dep version checking
```

Comment:
This is just for `check-dep`, right? Since we control that I think it’d be nice to have a cleaner interface for this than using the empty string as sentinel.